code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -1539,14 +1538,14 @@ func (rm *ReadMarker) VerifySignature(clientPublicKey string, balances chainstat
 
 func (rm *ReadMarker) verifyAuthTicket(alloc *StorageAllocation, now common.Timestamp, balances chainstate.StateContextI) (err error) {
 	// owner downloads, pays itself, no ticket needed
-	if rm.PayerID == alloc.Owner {
+	if rm.ClientID == alloc.Owner {
```

Comment:
This would force the readMarker's client id to be the same as allocation owner. Anyway, as we discussed in slack that we will do the auth ticket verification in blobbers, so this code will be removed in the future perhaps. But before that we can remove the `rm.ClienttID == alloc.Owner` checking.